db/migrate/20240808215701_add_column_metadata_to_saved_claims.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnMetadataToSavedClaims < ActiveRecord::Migration[7.1]
+  def change
+    add_column :saved_claims, :metadata, :string
```

Comment:
Can we store JSON in this column? Would a text be a better choice if it gets big?

Reply:
Updated 👍